src/components/dragelement/index.js

```diff
@@ -141,7 +141,12 @@ dragElement.init = function init(options) {
 
         if(!gd._dragged) {
             var e2 = document.createEvent('MouseEvents');
-            e2.initEvent('click', true, true);
+            e2.initMouseEvent('click', true, true,
```

Comment:
@etpinard Shall I keep `canBubble` and `cancelable` set to `true`? Or copy the the values from `e`?